File: BOJ/1934.cpp
#include<cstdio>
using namespace std;

int gcd(int a, int b) {
	if (a == 0) return b;
	return gcd(b%a, a);
}

int main() {
	int tc;
	scanf("%d", &tc);
	for (int i = 0; i < tc; i++) {
		int a, b, G;
		scanf("%d %d", &a, &b);
		if (a > b) G = gcd(b, a);
		else G = gcd(a, b);
		printf("%d\n", (a / G)*(b / G)*G);
	}

	return 0;
}

File: BOJ/python/1934.py
import sys
import math

input = sys.stdin.readline


def GCD(a, b):
    while b != 0:
        temp = b
        b = a % b
        a = temp

    return a


def main() -> None:
    T = int(input().rstrip('\n'))
    for _ in range(T):
        A, B = map(int, input().rstrip('\n').split(' '))

        # gcd = math.gcd(A, B)  # math 라이브러리
        gcd = GCD(A, B)  # 함수 구현
        lcm = abs(A * B) // gcd
        print(lcm)


if __name__ == "__main__":
    main()
